src/style.css

```diff
@@ -2,2 +1,89 @@
+:root {
+  --main-color: #00acdc;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
+  list-style: none;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

Comment:
Be consistent with your vertical margins. You're adding right margin to `.nav__item` which is horizontal, but make sure you apply the same principle to vertical margins as well, by adding only top or only bottom margin but not both.